src/bindings/js/node/src/tensor.cpp

```diff
@@ -138,3 +139,12 @@ Napi::Value TensorWrap::get_shape(const Napi::CallbackInfo& info) {
 Napi::Value TensorWrap::get_element_type(const Napi::CallbackInfo& info) {
     return cpp_to_js<ov::element::Type_t, Napi::String>(info, _tensor.get_element_type());
 }
+
+Napi::Value TensorWrap::get_size(const Napi::CallbackInfo& info) {
+    if (info.Length() > 0) {
+        reportError(info.Env(), "getSize() does not accept any arguments.");
+        return info.Env().Null();
```

Comment:
I propose return `info.Env().Undefined()` in case of error. (I know that in the sources you can meet both variants, but it could be aligned later).